app/uk/gov/hmrc/contactadvisors/FrontendAuditConnector.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.contactadvisors
+
+import javax.inject.{Inject, Singleton}
+import play.api.Mode._
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.play.audit.http.connector.{AuditConnector => Auditing}
+import uk.gov.hmrc.play.bootstrap.config.LoadAuditingConfig
+import uk.gov.hmrc.play.config.AppName
+
+@Singleton
+class FrontendAuditConnector @Inject()(val environment: Environment,
+                                       val conf: Configuration) extends Auditing with AppName {
```

Comment:
I don't believe you need this because the bootstrap audit connector is defined here: https://github.com/hmrc/bootstrap-play-25/blob/master/src/main/scala/uk/gov/hmrc/play/bootstrap/audit/DefaultAuditConnector.scala and wired here: 
https://github.com/hmrc/bootstrap-play-25/blob/master/src/main/scala/uk/gov/hmrc/play/bootstrap/AuditModule.scala and here: https://github.com/hmrc/message-frontend/blob/master/conf/application.conf#L18